content/docs/tasks/security/health-check/index.md

```diff
@@ -115,11 +127,14 @@ continuously failed health checks.
 
 ## Cleanup
 
-*   To disable health checking on Citadel, deploy Citadel with health checking disabled by setting the Helm argument `security.healthCheckEnabled` as `false`.
```

Comment:
this content should not be lost.  Please revert this part of the change.

Reply:
@ymesika my bad, thought we did :)  It must have been removed.

/lgtm
/approve